### PR TITLE
Heroku test

### PR DESCRIPTION
**Pull request by Luksack**
wtf am I doing :D

### CODE REVIEW

**Comment by Luksack** [comment]
Radek pls move

File: .idea/getimg.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Riglanto) [inline .idea/getimg.iml]: Ignore `.idea` directory while commiting.

File: Procfile [hunk 2]
```diff
@@ -0,0 +1 @@
+web: gunicorn app:app --log-file=-
```
Comment (Riglanto) [inline Procfile]: `--log-file` is probably redundant while you're passing no file

File: app.py [hunk 3]
```diff
@@ -11,8 +15,8 @@ def home_page():
 @app.route('/', methods=['POST', 'GET'])
```
Comment (Riglanto) [inline app.py]: I think the only method used here is `POST`. Even more, the default methods is probably `GET` so you have defined it twice :)

**Review by Riglanto — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,4 @@
 *.jpg
 *.pyc
-*.xml
\ No newline at end of file
+*.xml
+.idea/getimg.iml
```
Comment (Riglanto) [inline .gitignore]: Ignore whole `.idea` directory

File: .idea/getimg.iml (hunk 1 above)
Comment (Riglanto) [reply .idea/getimg.iml]: Do not commit this file!:P

**Review by Riglanto — CHANGES_REQUESTED** [review]

**Review by Riglanto — APPROVED** [review]

**Pull request merged**